setup.cfg

```diff
@@ -0,0 +1,38 @@
+[metadata]
+name = openfe
+version = 0.0.1.dev0
+description = 
+long_description = file: README.md
+long_description_type = text/markdown
+#author =
+#author_email =
+license = MIT
+license-file = LICENSE
+url = https://github.com/OpenFreeEnergy/openfe
+classifiers =
+    Development Status :: 1 - Planning
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: MIT License
+    Operating System :: POSIX
+    Operating System :: Microsoft :: Windows
+    Programming Language :: Python :: 3
```

Comment:
I generally tend to throw in separate supported minor versions here, but I guess it doesn't matter.